src/macros.jl

```diff
@@ -156,29 +156,47 @@ function parse_one_operator_constraint(_error::Function, vectorized::Bool,
     return parse_code, _build_call(_error, vectorized, variable, set)
 end
 
+function rewrite_call_expression(_error::Function, head::Val{F}, args...) where F
```

Comment:
Could this part be moved to a separate PR ?

Reply:
Here it is: https://github.com/JuliaOpt/JuMP.jl/pull/2229